themes/theme-gmd/pages/Category/components/CategoryListContent/index.jsx

```diff
@@ -49,12 +51,14 @@ class CategoryListContent extends PureComponent {
                 <CategoryList
                   categories={categories}
                   prerender={categoriesFetching ? childrenCount : 0}
+                  showImages={this.props.showCategoryImages}
```

Comment:
This does not work yet since `CategoryList` does not use the `showImages` prop. It needs to be added to the component as well. 
So inside of CategoryList you need to add something like:
` rightComponent={showImages ? (<CategoryImage className={image} src={category.imageUrl} : null />)}`


Testing of the code change would have shown it that it can't work like this ;)